### ISSUE TITLE
ut is missing

### ISSUE THREAD

**Issue by prudhomm** [feelpp/feelpp-tutorial-dev#12]
@lemoinep you mentioned that ut.hpp is missing
it is in fact here : https://github.com/feelpp/feelpp-tutorial-dev/blob/master/docs/modules/ROOT/examples/ut.hpp

**Comment by lemoinep** [comment]
> @lemoinep you mentioned that ut.hpp is missing it is in fact here : https://github.com/feelpp/feelpp-tutorial-dev/blob/master/docs/modules/ROOT/examples/ut.hpp

Yes. I noticed it afterwards. I had forgotten this file in Ubuntu environment  when I started in April.
